telekom-workflow-engine/src/main/resources/changelog/db-changelog.xml

```diff
@@ -243,4 +243,13 @@
     </sql>
   </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+  <changeSet id="workflow-engine-2023.01.02.1" author="Martin Kabanen" dbms="postgresql">
+    <validCheckSum>8:01c48ac208ef47d50e23077d5e9d12c5</validCheckSum>
```

Comment:
this row shouldn't be needed, unless you already propagated the changeset to some machines and changed it later

Reply:
True, i just followed existing pattern, but removed it now.